apps/app/src/views/routes/Prescription.tsx

```diff
@@ -70,21 +72,49 @@ export const Prescription = () => {
 
   const isMobile = useBreakpointValue({ base: true, sm: false });
   const tableWidth = useBreakpointValue({ base: 'full', sm: '100%', md: '75%' });
+  const { colorMode } = useColorMode();
 
   return (
     <Page kicker="Prescription" header={prescription?.treatment.name} loading={loading}>
       <VStack spacing={4} fontSize={{ base: 'md', md: 'lg' }} alignItems="start" w="100%" mt={0}>
-        {loading ? (
-          <Skeleton width="130px" height="35px" borderRadius="md" />
-        ) : (
-          <Button
-            aria-label="New Order"
-            as={RouterLink}
-            to={`/orders/new?patientId=${patient.id}&prescriptionId=${id}`}
-          >
-            Create Order
-          </Button>
-        )}
+        <HStack>
+          {loading ? (
```

Comment:
These buttons (included create order), should only show if prescription is ACTIVE

Reply:
added disabling to both